minecraft_servers/abbelsmp/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+    "server_name": "abbelsmp",
+    "nice_name": "AbbelSMP",
+    "direct_ip": "play.abbelsmp.de",
+    "server_wildcards": [
+      "%.abbelsmp.de"

```

Comment:
Not a real wildcard
https://i.imgur.com/wIwIPum.png

Reply:
Removed